intl/conf.py

```diff
@@ -1,78 +1,15 @@
-import os
-import sys
-import time
-import ablog
+# Obtain shared config values
+import os, sys
 
-# Configuration file for the Sphinx documentation builder.
+sys.path.append(os.path.abspath(".."))
+sys.path.append(os.path.abspath("../.."))
 
-# -- Project information
-project = "Nuitka the Python Compiler"
-copyright = "%s, Kay Hayen and Nuitka Contributors" % time.gmtime().tm_year
-author = "Kay Hayen"
-release = version = ""
-
-# -- General configuration
-
-extensions = [
-    # Sphinx's own extensions
-    "sphinx.ext.autodoc",
-    "sphinx.ext.extlinks",
-    "sphinx.ext.intersphinx",
-    "sphinx.ext.mathjax",
-    "sphinx.ext.todo",
-#    "sphinx.ext.viewcode",
-    # External extensions
-    "sphinx_copybutton",
-    "sphinx_design",
-#    "sphinx_inline_tabs",
-    "sphinxcontrib.youtube",
-    "sphinx-favicon",
-    "sphinx_sitemap",
-    # Disabled, loads to slow and cannot be limited to needing pages.
-    # "sphinxcontrib.asciinema",
-    # Blog extension
-    "ablog",
-]
-
-intersphinx_mapping = {
-    "python": ("https://docs.python.org/3/", None),
-    "sphinx": ("https://www.sphinx-doc.org/en/master/", None),
-}
-intersphinx_disabled_domains = ["std"]
-
-templates_path = ["../doc/_templates", ablog.get_html_templates_path()]
+from shared_conf import *
 
 language = 'en'  #'zh_CN'
-locale_dirs = ['locales/']  # path is example but recommended.
```

Comment:
I thought it being below `intl` made perfect sense though?

Reply:
`locale_dirs = ['../locales/']` explicitly specifies the path of internationalization.